deployer/src/deployer/upload.py

```diff
@@ -242,6 +242,9 @@ def client(self):
         return boto3.client("s3")
 
     def get_key(self, build_directory, file_path):
+        if file_path.name == "index.html":
+            # This simplifies our Lambda@Edge function that tansforms URL's to S3 keys.
```

Comment:
I think you should explain it a bit more here. It's a critical hack and it's here to stay and future generations need to understanding. 
A critical point to explain is that we should remind that all lookups to S3 are always done via our Lambda@Edge code. Also, we prefer URLs like `/en-US/docs/Web` and not `/en-US/docs/Web/`. 
And perhaps remind the reader that in S3 you can have a key called `/prefix/en-us/docs/web` and it can have `content-type: text/html` etc even though it doesn't have the file extension. 
I don't know. Something something that explains what's obvious to you and me today but not to us in the future or to the next person who sees this rather baffling code.